src/test/java/tryCatchFinally/ArithmeticTest.java

```diff
@@ -0,0 +1,25 @@
+package tryCatchFinally;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ArithmeticTest {
+
+
+    @Test
+    void division() {
+        Assertions.assertThat(Arithmetic.divide(6, 3)).isEqualTo(2);
+        Assertions.assertThat(Arithmetic.divide(3, 3)).isEqualTo(1);
+    }
+
+    @Test
+    void divisionByZero() {
+        try {
+            Arithmetic.divide(6, 0);
+            fail("My method didn't throw when I expected it to");
```

Comment:
AssertJ has a fluent API to verify that allows verify exceptions conveniently.
Take a look at example:

```
@Test
public void testException() {
   assertThatExceptionOfType(IOException.class).isThrownBy(() -> { throw new IOException("boom!"); })
                                               .withMessage("%s!", "boom")
                                               .withMessageContaining("boom")
                                               .withNoCause();
}
```

So you woun't have to catch and swallow an exception, which is not quite a good practice :)

Reply:
Added this test =)